weather@mockturtl/files/weather@mockturtl/applet.js

```diff
@@ -582,7 +582,8 @@ MyApplet.prototype = {
       this._currentWeatherPressure.text = pressure + ' ' + _(this._pressureUnit)
 
       // location is a button
-      this._currentWeatherLocation.url = weather.link
+      let tmp = weather.link.split('*')
```

Comment:
This might break if the link property in the AJAX response doesn't have an asterisk in the string for some reason in the future. You could guard it with something like:

```js
let tmp = weather.link;
if (weather.link.indexOf('*') > -1) {
  tmp = weather.link.split('*')[1]
}
this._currentWeatherLocation.url = tmp;
```
Though its probably a new API that won't change for a while, is my guess.